modules/clightning.nix

```diff
@@ -111,6 +110,10 @@ in {
         while [[ ! -e ${cfg.dataDir}/bitcoin/lightning-rpc ]]; do
             sleep 0.1
         done
+        # give group "search" access to allow clightning group processes to find lightning-rpc
```

Comment:
spark-wallet and lightning-charge need to be able to "look" into the directory to find `lightning-rpc`. Otherwise you get
```
Lightning client connection error { Error: connect EACCES /var/lib/clightning/bitcoin
```
in spark-wallet for example.